pl.js

```diff
@@ -0,0 +1,688 @@
+windows.publish.map.header Opublikuj lub zaaktualizuj grę
+windows.publish.map.thumbnail Bez miniaturki
+windows.publish.map.paste Wklej dane gry
+windows.publish.map.update Opublikuj/Zaaktualizuj
+windows.publish.map.login aby tworzyć i udostępniać gry!
+windows.theatre.header Tryb teatru
+windows.theatre.record Włącz nagrywanie
+windows.theatre.key Klawisz nagrywania
+windows.clans.header Klany
+windows.clans.members Członkowie ({0})
+windows.clans.requests Prośby dołączenia
+windows.clans.requests.none Brak próśb dołączenia
+windows.clans.page [{0}] strona klanu
+windows.clans.create Stwórz klan
+windows.clans.name Nazwa klanu
+windows.clans.view Pokaż klany
+windows.clans.join Dołącz do klanu
+windows.clans.login aby tworzyć lub dołączyć do klanu!
+windows.store.header Sklep
+windows.store.spin {0} Kręć
+windows.store.agree Kupując zgadzasz się z {0}Warunkami
+windows.store.purchase Kup KR
+windows.store.voucher Wykorzysta Voucher
+windows.store.voucherinfo Zdobądź vouchery od streamerów i w zdarzeniach
+windows.store.market Market
+windows.store.market2 Krunker Market
+windows.store.premium Kup Premium
+windows.store.premium2 Krunker Premium
+windows.store.buyP Wybierz wyświetlaną nazwę i więcej z
+windows.store.buy Kupuj i sprzedawaj przedmioty w
+windows.store.wheels Koła nagród
+windows.store.unlock Odblokuj nowe skiny i przedmioty
+windows.store.amount {0} KR na koncie
+windows.store.login aby kupować i odblokowywać nowe przedmioty w sklepie!
+windows.store.purchasing Kupowanie...
+windows.store.unboxed Odpakowano {0}{1}
+windows.trade.header Handlowanie
+windows.scope.header Wybierz lunetę
+windows.premium.header Premium
+windows.skin.header Wybierz skin
+windows.hat.header Wybierz kapelusz
+windows.back.header Wybierz tył
+windows.waist.header Wybierz talię
+windows.face.header Wybierz twarz
+windows.melee.header Wybierz broń białą
+windows.attach.header Wybierz dodatek
+windows.reticle.header Wybierz celownik
+windows.reticle.unlocked Odblokowane na poziomie {0}
+windows.reticle.req Wymagany poziom {0}
+windows.dye.header Wybierz barwnik
+windows.dye.default Domyślny barwnik
+windows.dye.spin Barwnik
+windows.changelog.header Lista zmian
+windows.mods.community.header Modyfikacje społeczeństwa
+windows.mods.community.fav Twoje ulubione
+windows.mods.community.new Nowe modyfikacje
+windows.mods.community.hot Popularne teraz
+windows.mods.community.name Nazwa modyfikacji
+windows.mods.community.none Nie znaleziono modyfikacji
+windows.publish.mod.header Opublikuj modyfikacje
+windows.publish.mod.thumbnail Bez miniaturki
+windows.publish.mod.name Nazwa modyfikacji
+windows.publish.mod.paste URL modyfikacji
+windows.publish.mod.update Opublikuj/Aktualizuj
+windows.publish.mod.login aby tworzyć i udostępniać modyfikacje!
+windows.secondary.header Wybierz drugą broń
+windows.secondary.unlocked Odblokowane
+windows.secondary.req Wymaga poziomu {0}
+windows.advertise.header Reklamuj
+windows.support.header Wsparcie
```

Comment:
It's a window header, it should be left at "Wsparcie", the previous one was set to "Reklamuj" because I think it could be a bit unclear if it was set to "Reklama"